### DISCUSSION TITLE
Bump ojs-runtime to 6.0.0

### DISCUSSION THREAD

**Discussion by hrbrmstr** [quarto-dev/quarto-cli#7474] [Feature Requests]
### Description

the latest ~runtime~ parser is 6.0.0 (https://cdn.skypack.dev/@observablehq/parser@6.0.0)

~I've got a local dev-mode quarto running with it and it seems to be fine.~

There are many new enhancements from recent Observable improvement that'd be nice to have in Quarto 1.4

**Comment by cscheid** [comment]
Do note that you linked to the observablehq parser; the [runtime is at 5.9.4](https://cdn.skypack.dev/@observablehq/runtime) and the [stdlib is at 5.8.3](https://cdn.skypack.dev/@observablehq/stdlib). We have to be a little careful on updates because we run a fork and we need to test. With that said, I have an open PR with other dependency updates and it's natural to add this as well. Keep an eye on https://github.com/quarto-dev/quarto-cli/pull/7419.

**Reply by cscheid** [reply]
In the meantime, the stdlib dependency changes are:

````
<<<<<<< HEAD
export const d3 = dependency("d3", "7.8.4", "dist/d3.min.js");
export const inputs = dependency("@observablehq/inputs", "0.10.4", "dist/inputs.min.js");
export const plot = dependency("@observablehq/plot", "0.6.5", "dist/plot.umd.min.js");
=======
export const d3 = dependency("d3", "7.8.5", "dist/d3.min.js");
export const inputs = dependency("@observablehq/inputs", "0.10.6", "dist/inputs.min.js");
export const plot = dependency("@observablehq/plot", "0.6.11", "dist/plot.umd.min.js");
>>>>>>> e5a4aa79c45de2219ae1d5414a234eb9a0b8bd8d
````

If you need to run the latest Plot or Inputs, you can import them directly from skypack too: https://cdn.skypack.dev/@observablehq/inputs and https://cdn.skypack.dev/@observablehq/plot.

````
---
title: test
---

```{ojs}
SkypackPlot = await import("https://cdn.skypack.dev/@observablehq/plot")
SkypackPlot.plot
```
````

<img width="695" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/285675/5bdb0cdb-5474-44bb-92ed-c9dbfcc1b129">

**Reply by hrbrmstr** [reply]
super helpful. I totally didn't even think to just use my own imports

**Reply by jimjam-slam** [reply]
Is SkyPack the best way to get imports? The website and API are currently down (I can't search for Arquero, for example) and look like [they have been down for months](https://status.skypack.dev).

**Reply by cscheid** [reply]
It's either that or esm.sh, and esm.sh has miscompiled some libraries in the past in my experience.
